### PR TITLE
Added Hack implementation

### PR DESCRIPTION
**Pull request by da-strange-boi**
## Description
An implementation of the sieve using the [Hack language](https://hacklang.org/)

## Contributing requirements
* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: PrimeHack/solution_1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# hhvm docker images only support amd64
+FROM hhvm/hhvm:latest
```
Comment (rbergen) [inline PrimeHack/solution_1/Dockerfile]: Please pin the base image to a specific version.

File: PrimeHack/solution_1/PrimeHack.hack [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            }
+        }
+        if ($showResults)
+            echo "\n";
+
+        $avg = ($duration / $passes) / 1000;
+        $timeInSecs = $duration / 1000;
+        echo "Passes: {$passes}, Time: {$timeInSecs}, Avg: {$avg}, Limit: {$this->sieveSize}, Count: {$count}, Valid: {$this->validateResults()}\n";
+        echo "\n";
+        echo "da-strange-boi;{$passes};{$timeInSecs};1;algorithm=base,faithful=yes,parallel=no,bits=unknown\n";
```
Comment (rbergen) [inline PrimeHack/solution_1/PrimeHack.hack]: Please remove the `bits` tag if the bit count per prime is unknown. The tag's absence itself is the indication of this.

File: PrimeHack/solution_1/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+### Hack
+1. Install HHVM runtime from the [offical download page](https://docs.hhvm.com/hhvm/getting-started/getting-started)
+2. Run the code using the follow syntax: `hhvm PrimeHack.hack`
+
+## Output
+Intel Core i5-2500 @ 3.30GHz (Ubuntu 20.04)
+```
+Passes: 51, Time: 5.011, Avg: 0.098254901960784, Limit: 1000000, Count: 78498, Valid: 1
+
+da-strange-boi;51;5.011;1;algorithm=base,faithful=yes,parallel=no,bits=unknown
```
Comment (rbergen) [inline PrimeHack/solution_1/README.md]: I'm not going to withhold my approval of the PR over this, but you may want to remove the `bits` tag from here as well.

**Review by rbergen — APPROVED** [review]
LGTM

**Review by marghidanu — APPROVED** [review]

**Pull request merged**